style.css

```diff
@@ -235,120 +532,630 @@ body {
+  }
+
+  .order__title {
+    font-size: 37px;
+    line-height: 38px;
+  }
+
+  .order__subtitle {
+    font-size: 15px;
+    line-height: 15px;
+    margin-bottom: 4vw;
+  }
+
+  .order__description {
+    font-size: 14px;
+    line-height: 20px;
+    margin-bottom: 3vw;
+  }
+
+  .color-list {
+    height: 33px;
+    margin-bottom: 4.5vw;
+    margin-right: 1vw;
+  }
+
+  .list__item {
+    margin-right: 0.5vw;
+  }
+
+  .list__item_grey,
+  .list__item_blue,
+  .list__item_green {
+    width: 27.18px;
+    height: 27px;
+  }
+
+  .list__item_pink {
+    width: 33px;
+    height: 33px;
+  }
+
+  .order__price {
+    font-size: 37px;
+    line-height: 55px;
+    transform: translateY(1px);
+  }
+
+  .order-number {
+    width: 95px;
+    height: 40px;
+    margin-bottom: 2.8vw;
+    margin-right: 1vw;
+    padding-left: 8px;
+    padding-top: 10px;
+  }
+
+  .number__input,
+  .minus,
+  .plus {
+    font-size: 14px;
+  }
+
+  .minus {
+    margin-left: 0.5vw;
+    margin-right: 1.2vw;
+  }
+
+  .number__input {
+    margin-right: 1vw;
+  }
+
+  .order__cart {
+    font-size: 12px;
+    margin-bottom: 4vw;
+    height: 40px;
+    padding-right: 0.7vw;
+    transform: translateY(-2px);
+    width: 220px;
+  }
+
+  .range-1 {
+    transform: translateX(2px);
+  }
+
+  .range-1__link {
+    font-size: 12px;
+    line-height: 15px;
+    margin-right: 1vw;
+  }
+
+  .range-1__link-arrow {
+    width: 220px;
+  }
+
+  .outfit {
+    padding: 10.5vw 12vw 9vw 7.5vw;
+  }
+
+  .outfit__img {
+    height: 70%;
+    width: 75%;
+  }
+
+  .outfit-details {
+    margin-top: 5vw;
+  }
+
+  .outfit-content {
+    margin-bottom: 3vw;
+    margin-right: 0;
+    padding: 1.5vw 3vw 1.2vw 4.5vw;
+    transform: translateX(-175px);
+    width: 460px;
+  }
+
+  .outfit__title {
+    font-size: 35px;
+    line-height: 36px;
+  }
+
+  .outfit__subtitle {
+    font-size: 15px;
+    line-height: 15px;
+    margin-bottom: 3vw;
+  }
+
+  .outfit__description {
+    font-size: 14px;
+    line-height: 21px;
+    margin-bottom: 3.5vw;
+  }
+
+  .outfit__cart {
+    font-size: 12px;
+    height: 40px;
+    width: 220px;
+    transform: translateY(-1px);
+  }
+
+  .range-2 {
+    transform: translateX(15px);
+  }
+
+  .range-2__link {
+    font-size: 12px;
+    line-height: 15px;
+    margin-right: 8px;
+  }
+
+  .range-2__link-arrow {
+    width: 70px;
+  }
+}
+
+@media (max-width: 1024px) {
+  .header__logo {
+    margin-right: 7vw;
+  }
+
+  .language {
+    display: none;
+  }
+
+  .collection {
+    padding-top: 45px;
+  }
+
+  .collection__title {
+    font-size: 46px;
+    line-height: 46px;
+    margin-bottom: 2vw;
+    margin-top: 4vw;
+  }
+
+  .collection__description {
+    font-size: 14px;
+    line-height: 20px;
+    margin-bottom: 2.5vw;
+  }
+
+  .collection__btn {
+    font-size: 12px;
+    margin-right: 2.5vw;
+  }
+
+  .range__link {
+    color: #000;
+    font-size: 12px;
+    margin-right: 1vw;
+  }
+
+  .scroll {
+    display: none;
+  }
+
+  .order {
```

Comment:
Codacy found an issue: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)